pkg/cache/cache.go

```diff
@@ -77,6 +78,13 @@ func WithExcludedResourcePrefixes(excludedPrefixes []string) Option {
 	}
 }
 
+// WithResourceMappings sets the resource transformer
+func WithResourceMappings(transforms []config.ResourceTransformation) Option {
```

Comment:
```suggestion
func WithResourceTransformations(transforms []config.ResourceTransformation) Option {
```
nit, for consistency